src/roadmap/release-checklist.md

```diff
@@ -34,47 +36,92 @@ A release consists of the following artifacts:
 images referenced above. For example, if `$WORKFLOW_RELEASE` is `2.0.0-rc1`, the new chart would
 be in a new directory called `workflow-rc1`.
 
-# Step 1: Create New Helm Classic Charts
+# Step 1: Cut repo branches and push image tags
 
-First, export necessary values for `WORKFLOW_RELEASE` and `WORKFLOW_RELEASE_SHORT`:
-```
-export WORKFLOW_RELEASE=<full release name>
-export WORKFLOW_RELEASE_SHORT=<short form of above>
-```
+(For actions below, [sgoings/deis-workflow-group](https://github.com/sgoings/deis-workflow-group) was used)
+
+1. Once the release milestone is cleared of tickets in the workflow component repos, the release branches can be cut.  
+
+  If only a particular repo is ready, navigate to said repo and:
+  ```
+  git checkout master && git pull origin master
+  git checkout -b release-$WORKFLOW_RELEASE && git push origin release-$WORKFLOW_RELEASE
```

Comment:
its funny you say that. i specifically rename my origin to be deis so i dont accidentally do `git push origin master` (even though we have branch protection)